webpack/components/extensions/HostDetails/Tabs/ErratumExpansionContents.js

```diff
@@ -0,0 +1,61 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import {
+  TreeView,
+} from '@patternfly/react-core';
+import { translate as __ } from 'foremanReact/common/I18n';
+
+const ErratumExpansionContents = ({ erratum }) => {
+  const {
+    bugs, cves, packages,
+    module_streams: moduleStreams,
+  } = erratum;
+  const [activeItems, setActiveItems] = useState(null);
+  const options = [
+    {
+      name: __('Bugs'),
+      id: 'bugs',
+      children: bugs.map(bug => ({ name: bug.bug_id, id: bug.bug_id, ...bug })),
```

Comment:
children: bugs.map(({bug_id:bugId, ...rest}) => ({ name: bugId, id: bugId, ...rest })),

Reply:
can we destructure where possible?

Reply:
I don't think it's possible here because the input object has only `bug_id` and the resulting object must have keys `name` and `id`.  or maybe I'm missing something?

Reply:
Oh I see, yes I get why you did it your way now.